editor/components/rich-text/tinymce.js

```diff
@@ -96,6 +182,23 @@ export default class TinyMCE extends Component {
 		} );
 	}
 
+	saveEditorNode( editorNode ) {
+		this.editorNode = editorNode;
+
+		/**
+		 * A ref function can be used for cleanup because React calls it with
+		 * `null` when unmounting.
+		 */
```

Comment:
Oh, that's cool!

Reply:
Yeah, I love being able to keep allocation and cleanup in one place while also handling real situations where a ref is removed and re-added during the life of a component.

Reply:
Shouldn't we just use `componentDidMount` and `componentWillUnmount` ?

Related: The latest version of React includes a new `React.createRef` function which may become the recommended standard in place of the `ref` callback:

https://reactjs.org/docs/refs-and-the-dom.html#creating-refs